tools/test.py

```diff
@@ -317,7 +317,7 @@ def HasRun(self, output):
 class ActionsAnnotationProgressIndicator(DotsProgressIndicator):
   def AboutToRun(self, case):
     case.additional_flags = case.additional_flags.copy() if hasattr(case, 'additional_flags') else []
-    case.additional_flags.append('--test-reporter=./tools/github_reporter/index.js')
+    case.additional_flags.append('--test-reporter=./test/common/test-error-reporter.js')
```

Comment:
This is actually not available yet, so we would need an intermediate value or waiting for  #56438 and rebasing

Reply:
If you didn't want to block on #56438, you could delete this in this PR and add the reporter from #56438 in that PR.

Reply:
I don't mind waiting on the other PR, unless you want to move this fast

Reply:
Now that #56438 has landed, we can move forward with this.